test/Abp.EntityFrameworkCore.Tests/Tests/Repository_Filtering_Tests.cs

```diff
@@ -106,5 +108,20 @@ public async Task Should_Filter_MustHaveTenantId()
             //TODO: Create unit test
             //TODO: Change filter
         }
+
+        [Fact]
+        public async Task Should_Filter_View_With_MustHaveTenantId()
+        {
+            //Should get all entities for the host
+            var ticketsDefault = await _ticketListItemRepository.GetAllListAsync();
```

Comment:
I think @thomas-parrish followed the approach in Should_Filter_MustHaveTenantId test in the same class. But, you are right, using this is better. Since the PR is merged, I will do that in dev branch.